### PR TITLE
DEP Upgrade frontend build stack

### PR DESCRIPTION
**Pull request by GuySartorelli**
See https://github.com/silverstripe/silverstripe-admin/pull/1389 for a wider discussion on most of the changes

## Parent issue
- https://github.com/silverstripeltd/product-issues/issues/644

### CODE REVIEW

File: client/src/components/TOTP/Register.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import QRCode from 'qrcode.react';
+import { QRCodeSVG } from 'qrcode.react';
```
Comment (ChloeSartorelli) [inline client/src/components/TOTP/Register.js:5]: The old `QRCode` component was deprecated - see https://github.com/zpao/qrcode.react#qrcode---deprecated

**Review by emteknetnz — CHANGES_REQUESTED** [review]
behat failures

**Comment by GuySartorelli** [comment]
Behat failures in this PR are expected - it relies on various other PRs. Refer to the overall kitchen sink CI in the issue card

**Review by emteknetnz — APPROVED** [review]

**Comment by emteknetnz** [comment]
Was tested manually

**Pull request merged**